CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Firefly committers
+* @hyperledger-labs/firefly-committers
```

Comment:
Is this an automatically made team based on who has committed to FF? Or is this something admins have control over?

Reply:
This is the list of maintainers in the Hyperledger Labs GitHub Org.